update ReadMe + index.html (title)

vSt11 opened the pull request:
Update readme with MERN resources
Update index.html title to wipe out REACT APP and add custom title to the main page

atxr commented on frontend/public/index.html:
```diff
@@ -24,7 +24,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>ISCSC - Site du club informatique ISAE-Supaero</title>
```
Two things here:
- You should respect the case :wink: 
```diff
- ISCSC
+ iScsc
```
- An english title would be better to be consistent :thinking:

atxr reviewed (changes requested):
Thanks for you first contribution!
I just suggested a change for the website title

amtoine reviewed (approved):
given the change requested by @atxr -with which i agree!-, i approve this PR :+1: 

thanks a lot for the contribution :relieved:

ctmbl reviewed (commented):
I agree with @amtoine: @atxr changes are enough.  
I'll add two things however:
 - here the changes are **really** minor so never mind but in future PR try to address one change at a time :wink: 
 - great resources you linked here! I added these to the [wiki](https://github.com/iScsc/iscsc.fr/wiki/First-steps-and-resources#resources) so tks a lot! and I think that once the wiki will be clean/complete enough we'll simply add a link to it in that section of the README
 
 Once again tks for your contribution!

vSt11 commented:
I think I FINALLY got the workflow, as you can see it was not easy lol
To sum this up : i changed the case correctly for iScsc, and added some upper case to first letters of the title that i translated in english

amtoine reviewed (approved):
the title looks fine :ok_hand: 

thanks :+1:

ctmbl reviewed (approved):
@vSt11 
I think we'll need to give you a quick reminder on how to make commits and pushes :rofl: 
As stated in private by @amtoine your commits are quite... artistic! Jokes apart we'll make a quick call sometimes to talk about it :+1: 
However the diff looks good so I think we can merge this one!

ctmbl commented:
@atxr this one also is waiting for you :joy:

atxr reviewed (approved):
Sorry for the delay!
That looks good :fire: 
I'll let you merge @vSt11 :rocket:
